packages/editor-ui/src/views/NodeView.vue

```diff
@@ -1848,6 +1849,26 @@ export default mixins(
 				this.setZoomLevel(1);
 				setTimeout(() => {
 					this.$store.commit('setNodeViewOffsetPosition', {newOffset: [0, 0]});
+					// For novice users (onboardingFlowEnabled == true)
+					// Inject welcome sticky note and zoom to fit
+					if(newWorkflow.onboardingFlowEnabled) {
+						this.$nextTick(async () => {
+							await this.addNodes([
+								{
+									...CanvasHelpers.WELCOME_STICKY_NODE,
+									parameters: {
+										...CanvasHelpers.WELCOME_STICKY_NODE.parameters,
+										content: this.$locale.baseText(
+											'onboardingWorkflow.stickyContent',
+											{
+												interpolate: { imgURL: `/static/n8n-quickstart-v5-thumbnail.webp` },
+											}),
```

Comment:
should not we include imgUrl in translation key as well? that way embedders can replace image too?